epub33/a11y-tech/index.html

```diff
@@ -1680,6 +1690,338 @@ <h4>Identify the pagination source</h4>
+							for groups of footnotes and individual footnotes, respectively.</li>
+						<li>Page breaks &#8212; use the <a data-cite="epub-ssv-11#pagebreak"><code>pagebreak</code>
+								semantic</a> [[EPUB-SSV-11]] for each page break.</li>
+					</ul>
+
+					<p>EPUB Creators may identify other structures but it is not necessary to meet this requirement.</p>
+
+					<aside class="example" title="Identifying a skippable footnote">
+						<p>In this example, the <code>footnote</code> semantic identifies a skippable footnote in the
+							Media Overlay Document.</p>
+
+						<p>Media Overlay Document:</p>
+
+						<pre>&lt;smil
+    xmlns="http://www.w3.org/ns/SMIL" 
+    xmlns:epub="http://www.idpf.org/2007/ops"
+    version="3.0">
+   &lt;body>
+      &lt;!-- a paragraph -->
+      &lt;par id="id1">
+         &lt;text
+             src="chapter1.xhtml#para1"/>
+         &lt;audio
+             src="chapter1_audio.mp3"
+             clipBegin="0:23:22.000"
+             clipEnd="0:24:15.000"/>
+      &lt;/par>
+
+      &lt;!-- a footnote -->
+      &lt;par
+          id="id2"
+          epub:type="footnote">
+         &lt;text
+             src="chapter1.xhtml#fn01"/>
+         &lt;audio
+             src="chapter1_audio.mp3"
+             clipBegin="0:24:15.000"
+             clipEnd="0:24:53.821"/>
+      &lt;/par>
+
+      &lt;!-- another paragraph -->
+      &lt;par id="id3">
+         &lt;text
+             src="chapter1.xhtml#para2"/>
+         &lt;audio
+             src="chapter1_audio.mp3"
+             clipBegin="0:24:53.821"
+             clipEnd="0:25:28.530"/>
+      &lt;/par>
+   &lt;/body>
+&lt;/smil>
+</pre>
+
+						<p>EPUB Content Document:</p>
+
+						<pre>&lt;html … >
+   …
+   &lt;body>
+      &lt;p id="para1">
+         This is the paragraph with the footnote reference …
+      &lt;/p>
+      
+      &lt;p id="fn01" role="doc-footnote">
+         This is the footnote …
+      &lt;/p>
+      
+      &lt;p id="para2">
+         This is the paragraph after the footnote …
+      &lt;/p>
+   &lt;/body>
+&lt;/html></pre>
+					</aside>
+				</section>
+
+				<section id="sync-004">
+					<h4>Identifying escapable structures</h4>
+
+					<p>Some content elements are containers for expressing complex information. A table, for example,
+						has data arranged in rows and cells, where each cell may contain several paragraphs of
+						information. Lists similarly may contain many items, and each item can contain complex
+						information. While users may be interested in some of the information in these structures, they
+						also often want to escape from them to keep reading, not have to listen to the entire content
+						before being able to move on. These are called escapable elements, because the user needs to be
+						able to escape from them whenever they choose to simplify the reading experience.</p>
+
+					<p>EPUB 3's <a data-cite="epub-33#sec-media-overlays">Media Overlays feature</a> [[EPUB-33]] does
```

Comment:
I think it's more direct if the first sentence says:

"EPUB 3's Media Overlays feature only supports escapability if EPUB Creators add structural semantics to the markup."

Because at first glance it looks like we're saying that the features are not available in sync text/audio.